bazel/repository_locations.bzl

```diff
@@ -669,7 +669,7 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         urls = ["https://storage.googleapis.com/quiche-envoy-integration/{version}.tar.gz"],
         use_category = ["dataplane_ext"],
         extensions = ["envoy.transport_sockets.quic"],
-        last_updated = "2020-09-18",
+        release_date = "2020-09-18",
```

Comment:
This date is the date I updated the tar ball. The release date of this commit should be 2020-09-17